### PR TITLE
set config defaults and update example configmap

### PR DESCRIPTION
**Pull request by auhlig**
if the bind, metrics section is not defined in the configmap, set the defaults.

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ import (
 
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
+	"strings"
```
Comment (mrIncompetent) [inline config.go]: wrong order

**Review by mrIncompetent — CHANGES_REQUESTED** [review]
Except for the imports: lgtm

**Comment by mrIncompetent** [comment]
Sorry for the huge delay - completely missed the notification

**Comment by auhlig** [comment]
Rearranged the imports @mrIncompetent

File: config.go [hunk 2]
```diff
@@ -29,6 +29,8 @@ import (
 
 	"github.com/golang/glog"
 
+	"strings"
```
Comment (mrIncompetent) [inline config.go]: Should be at the top where all the golang-sdk import are

Comment (auhlig) [reply config.go]: That's the order `goimports` suggests. Anyway I manually did it.

Comment (mrIncompetent) [reply config.go]: Thats because strings was in a separate block. goimports sorts block-wise :)

**Comment by mrIncompetent** [comment]
Thank you very much for the PR :)

**Review by mrIncompetent — APPROVED** [review]

File: config.go (hunk 2 above)
Comment (sttts) [reply config.go]: Somebody should fork goimports to implement the kube import order conventions. k8s.io/kimport.

**Pull request merged**